tests/ert_tests/dark_storage/conftest.py

```diff
@@ -4,8 +4,8 @@
 import shutil
 from argparse import ArgumentParser
 
-import py
 import pytest
+from py import path
```

Comment:
Hadn't heard about `py` before (https://pypi.org/project/py/), but they in fact explicitly states that one should use `pathlib` instead:
`py.path: uniform local and svn path objects -> please use pathlib/pathlib2 instead`

If you have the time please consider updating with `pathlib` :)